### DISCUSSION TITLE
Recommend go library: A Lodash-style Go library based on Go 1.18+ Generics (map, filter, contains, find...)

### DISCUSSION THREAD

**Discussion by jintaiyong** [avelino/awesome-go#4916] [Ideas]
I hope you guys who is in charge of here can consider adding this library(https://github.com/samber/lo) to https://github.com/avelino/awesome-go#stream-processing

**Comment by avelino** [comment]
to propose new projects you need to open pull requests including them, remembering that there must be a pull request for each project

[I recommend reading](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md)
